lighthouse-core/report/templates/report-template.html

```diff
@@ -247,8 +249,14 @@ <h2 class="config-section__title">Blocked URL Patterns</h2>
 {{/each}}
 
 <script>
-window.addEventListener('DOMContentLoaded', _ => {
-  new LighthouseReport(window.lhresults);
+(function(exports) {
```

Comment:
oh, is it because we statically include them and `brfs` inlines them? `brfs` doesn't have an explicit `ignore` option, but we could probably add that to cut down on the viewer bundle